client/python/samples/README.md

```diff
@@ -92,6 +101,7 @@ optional arguments:
   --grpc_port GRPC_PORT Specify port to grpc service. default: 9000
   --model_name MODEL_NAME Model name to query. default: resnet
   --model_version MODEL_VERSION Model version to query. Lists all versions if omitted
```

Comment:
We need to correct this. Metadata call with none or 0 version returns result for latest version, not all.